DNN Platform/Library/Entities/Urls/RewriteController.cs

```diff
@@ -1392,7 +1392,11 @@ internal static void GetUrlWithQuerystring(HttpRequest request, Uri requestUri,
                 absoluteUri = absoluteUri.Replace(queryString, "");
             }
             absoluteUri = HttpUtility.UrlDecode(absoluteUri); //decode the incoming request
-            string rewritePath = GetTabFromDictionary(absoluteUri, queryStringCol, settings, result, parentTraceId);
+
+            // below 810 check is taking care of lanugage code
+            var absoluteUriWithoutLanguageCode = GetUrlWithoutLanguageCode(absoluteUri, result.CultureCode);
+            
+            string rewritePath = GetTabFromDictionary(absoluteUriWithoutLanguageCode, queryStringCol, settings, result, parentTraceId);
```

Comment:
By making the change in the way that you get the re-write path doesn't this change the actual content that is obtained?  It might be ok, but I want to make sure that it is correct. before we go.

Also, this change has a potential to break the API for users that have worked around.  I would prefer to NOT include this breaking change in 9.3, but will defer to the rest of the @dnnsoftware/tag

Reply:
@mitchelsellers  In fact this change, is intended to correct the content, when multilingual option is enable. If you check #2335, TransformFriendlyUrlToQueryString is getting extra or additional parameter of language code, which is the problem or issue.

Reply:
@dnnsoftware/tag Thoughts?  I'm not ok with this.

Reply:
This can indeed be a breaking change if people are not following the practice pointed out by @sleupold above.  This doesn't sound like a good idea for 9.3 IMO.